### PR TITLE
Fixes to #1898

### PR DESCRIPTION
**Pull request by wojciechtanski**

### CODE REVIEW

**Comment by robotci** [comment]
Admin, can CI be enabled?

File: atest/robot/standard_libraries/reserved.robot [hunk 1]
```diff
@@ -13,3 +13,9 @@ Continue
 
 Reserved in User Keyword
     Check testcase    Reserved In User Keyword
+
+Else should be capitalized
+    Check testcase    Else should be capitalized
+
+Else If should be capitalized
+    Check testcase    Else If should be capitalized
```
Comment (pekkaklarck) [inline atest/robot/standard_libraries/reserved.robot]: Newline still missing.

File: src/robot/libraries/Reserved.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#  Copyright 2008-2015 Nokia Solutions and Networks
+# Copyright 2008-2015 Nokia Solutions and Networks
```
Comment (pekkaklarck) [inline src/robot/libraries/Reserved.py]: One space removed here.

**Comment by pekkaklarck** [comment]
Few minor whitespace issues still. Could you please fix them and then [squash commits](http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit)? The latter isn't absolutely necessary in this case, but you may want to practice it with a small PR.

**Comment by pekkaklarck** [comment]
@robotci: Whitelist

**Comment by wojciechtanski** [comment]
@pekkaklarck is it possible to squash commits if i did not have separate branch on my fork? i was working directly on master.

**Comment by pekkaklarck** [comment]
I think it should be. Branches are generally a good idea, though.

**Comment by pekkaklarck** [comment]
Is this or #2001 currently active? Could you close the one that's not?

**Pull request closed without merging**